src/Diagnostics.RuntimeHost/Services/StorageService/StorageService.cs

```diff
@@ -39,11 +42,13 @@ public class StorageService : IStorageService
         private bool loadOnlyPublicDetectors;
         private bool isStorageEnabled;
         private CloudTable cloudTable;
+        private string configurationTable;
 
         public StorageService(IConfiguration configuration, IHostingEnvironment hostingEnvironment)
         {
             tableName = configuration["SourceWatcher:TableName"];
             container = configuration["SourceWatcher:BlobContainerName"];
+            configurationTable = configuration["SourceWatcher:DiagConfiguration"];
```

Comment:
Could we add these to the appsettings? Just so that we have a one source of truth of all application settings needed for the runtimehost.

Reply:
you mean to appsettings.json?  I thought of adding it directly to keyvault

Reply:
Having it in both I think is good